app/helpers/application_helper.rb

```diff
@@ -1,17 +1,17 @@
 module ApplicationHelper
   def thumbnail_url(image)
-    if image.is_video?
+    if image&.is_video?
```

Comment:
When would this be called without a valid image? I didn't need the `&` before.

Reply:
I don't exactly remember what caused it. Maybe your code had the check for the image in the erb file so the `thumbnail_url` was not being called for an artwork without an image.

Reply:
it happens when artwork doesn't have an image.
<img width="1042" alt="Screen Shot 2024-03-21 at 8 42 50 PM" src="https://github.com/netvvrk/members-database/assets/687513/ed4f66f7-3dd4-4f39-a9a9-9ce32046c1ee">